src/theory/subs_minimize.cpp

```diff
@@ -124,8 +262,6 @@ bool SubstitutionMinimize::find(Node n,
   Trace("subs-min") << "--- Compute relevant variables..." << std::endl;
   std::unordered_set<Node, NodeHashFunction> rlvFv;
   // only variables that occur in assertions are relevant
```

Comment:
Does this comment still belong here?

Reply:
Yes, relevant here means "possibly required or implied".